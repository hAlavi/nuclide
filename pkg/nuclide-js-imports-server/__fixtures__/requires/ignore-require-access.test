require.requireActual('./polyfill/iterator.enumerate.js');

const x = X;
