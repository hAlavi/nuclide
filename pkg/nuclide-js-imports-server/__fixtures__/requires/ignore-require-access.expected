require.requireActual('./polyfill/iterator.enumerate.js');

const X = require('X');

const x = X;
